pom.xml

```diff
@@ -1006,6 +1010,10 @@
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <excludeGroups>${accumulo.integrationTestsExcludedGroups}</excludeGroups>
+              <groups>${accumulo.integrationTestsGroups}</groups>
+            </configuration>
```

Comment:
At the very least, I've mentioned on several occasions the desire to use a category to replace `-Psunny`. I'm not so much concerned about updating docs, but obsoleting them. Documenting new categories as options is one thing. Adding whole new documention to execute these categories, different than the way these profiles are activated (or creating different combinations of groups each in new profiles) is much more problematic.